examples/directive/sink_mock.go

```diff
@@ -0,0 +1,86 @@
+package directive
+
+import (
+	sort3 "sort"
+	"sync/atomic"
+	"testing"
```

Comment:
`"sort"` gets aliased because it comes lexicographically after packages prefixed with `github.com`. This would also apply to `"sync/atomic"` and `"testing"` except that I've included logic for [prioritizing packages imported by the mock template itself](https://github.com/nicheinc/mock/blob/f381e7e3e997899e59c7462ecf005b79fba17633/iface/get.go#L169-L177). That saves us having to make `T *testing.T`, `atomic.AddInt32(&m.fCalled, 1)` etc. (which currently hard-code the package names into the template) aware of package renaming.